lib/analysers/taxons/grid_and_leaf_link_analyser.rb

```diff
@@ -11,10 +11,13 @@ def grid_results(taxon)
         taxon.body_html.css(CssSelector.for(:grid_taxon_links))
           .reduce([]) do |results, link|
 
+          links_total = link.ancestors(CssSelector.for(:grid_subsection))
+            .css(CssSelector.for(:grid_taxon_links)).count
+
           results << {
             taxon_base_path: taxon.base_path,
             link_href: link.attr('href'),
-            total_number_of_links: "N/A",
+            total_number_of_links: links_total,
             total_number_of_links_per_section: 'N/A',
```

Comment:
Hi @alecgibson ,

I suppose it does make sense to not display the column if it isn't really displaying any useful information. I have added a commit [here](https://github.com/alphagov/govuk-tagging-monitor/pull/18/commits/bd169ada08b76d8d4f3aae2f026c2c74a5448900) which results in [this spreadsheet](https://docs.google.com/spreadsheets/d/1x6ND04vHFBMyhJO66StdLEBNLJvjs1nkXB3koV8pR6o/edit#gid=362620695) (only displays `total_number_of_links_per_section` for accordions). Let me know what you think/how I could maybe improve it.